docs/current/cookbook/snippets/modules-shared-client/main.py

```diff
@@ -0,0 +1,17 @@
+import sys
+
+import anyio
+from pipelines import version
```

Comment:
Better to use a relative imports for local code:
```suggestion
from .pipelines import version
```